common/common_helper.sh

```diff
@@ -294,3 +350,13 @@ function check_minikube() {
 	fi
 }
 
+###########################################
+#   Check if kind is installed
+###########################################
+function check_kind() {
+	if ! which kind >/dev/null 2>/dev/null; then
+		echo "ERROR: Please install kind and try again"
+		print_min_resources
```

Comment:
Are the resource requirements same? Update the message in print_min_resources with cluster_type

Reply:
Yes, we're keeping it the same as `minikube`.
Updated the message now to make it cluster_type specific.